src/test/kotlin/org/rust/lang/core/resolve/RsPackageLibraryResolveTest.kt

```diff
@@ -526,6 +526,28 @@ class RsPackageLibraryResolveTest : RsResolveTestBase() {
         pub use foo_ as foo;
     """)
 
+    @UseNewResolve
+    @MockEdition(CargoWorkspace.Edition.EDITION_2018)
```

Comment:
Let's also add tests where a macro is defined:
- in the same mod
- in other mod in the same crate
- locally in a function body
And let's also test that we don't resolve macro 2.0 in lexical scope.